### PR TITLE
update wpCache

### PR DESCRIPTION
**Pull request by wpCache**
New version 2.0 Fix error browser chrome - firefox

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -10309,6 +10309,28 @@
...
+      "cats": [
+        23
+      ],
+      "headers": {
+        "X-Powered-By": "wpCache(?:/([\\d.]+))?\\;version:\\1"
+      },
+      "html": "<!--[^>]+wpCache",
+      "url": "^https?://[^/]+\\.wpcache\\.co",
+      "icon": "wpCache.png",
+      "implies": [
```
Comment (gadcam) [inline src/apps.json]: Does this mean you **need** both Apache and Nginx to run wpCache ?

Comment (wpCache) [reply src/apps.json]: Hello yes

https://wpcache.co/documentation/

== v2.0 (10.02.2018) ============
* Added: NGINX Settings
* Added: HTACCESS Auto Install
* Added: Restart Services PHP.ini
* Improved: Apache HTTP Server 2.4.x
* Improved: PHP 7.1.14
* Improved: nginx 1.13.8
* Improved: Varnish Cache 5.2.1
* Fixed: Dashboard Settings
* Fixed: wpCache.php
* Fixed: Major bug

Comment (gadcam) [reply src/apps.json]: I read the documentation where it is written
>Does wpCache® run with Apache, Nginx, Lighttpd...?
>Yes, wpCache® is compatible with Apache, Nginx, Microsoft IIS et
>LiteSpeed web servers.

I saw no mention of a strong requirement of Apache or Nginx so I think you should remove them from the implies field. :) (cf https://www.wappalyzer.com/docs/specification for the exact meaning of the implies field)

Comment (wpCache) [reply src/apps.json]: Requirements
PHP up 7.1.14
Apache ( NGINX+ / Varnish Cache )
Windows Server IIS
HTTP/1.0 or HTTP/2.0

look the core
https://github.com/wpCache/wpCache/tree/master/wpCache/core

Now if you are annoyed by anything you can change it please. thank you!

Comment (wpCache) [reply src/apps.json]: ok change implies

      "implies": [
        "WordPress",
        "PHP"
      ],

File: src/apps.json [hunk 2]
```diff
@@ -10309,6 +10309,26 @@
       "icon": "WHMCS.png",
       "website": "http://www.whmcs.com"
     },
+    "wpCache": {
+      "cats": [
+        23
+      ],
+      "headers": {
+        "X-Powered-By": "wpCache(?:/([\\d.]+))?\\;version:\\1"
```
Comment (jvoisin) [inline src/apps.json]: Can you add a `^` in the front of the regexp?

**Comment by jvoisin** [comment]
It seems that this application only has [3 stars]( https://github.com/wpCache/wpCache ) on github.

**Comment by wpCache** [comment]
new upload wpCache Version 1.9 free version    01.02.2018

**Comment by gadcam** [comment]
@jvoisin There are at least 5300 installations ;) cf https://www.wappalyzer.com/technologies/wpcache
To me, everything is ok now.

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**